charts/xrd-vrouter/values.schema.json

```
@@ -333,6 +333,14 @@
       "description": "CPU settings",
       "type": "object",
       "properties": {
+        "controlPlaneCpuset": {
+          "description": "CPUset for the XRd control plane to use",
```

```suggestion
          "description": "cpuset for the XRd control plane to use",
```

Doesn't really matter and could go either way, but all lowercase is consistent with elsewhere in this document.